packages/components/src/navigator/test/index.tsx

```diff
@@ -477,16 +478,27 @@ describe( 'Navigator', () => {
 			height: 100,
 		},
 	] );
+	// Mock `matchMedia` so that all animations are skipped.
+	const mockedMatchMedia = jest.fn( ( query: string ) => {
+		if ( /prefers-reduced-motion/.test( query ) ) {
+			return { matches: true } as ReturnType< typeof window.matchMedia >;
```

Comment:
Isn't it a better bet to test with all animations enabled? After all, there might be bugs that are only encountered with animations on.

Reply:
I don't think `js-dom` supports animations well — `testing-library` has even an [example](https://testing-library.com/docs/example-react-transition-group/#mock) of how to mock `React Transition Group`.